src/app/clusters/level-control/level-control.cpp

```diff
@@ -1255,14 +1314,13 @@ static void stopHandler(app::CommandHandler * commandObj, const app::ConcreteCom
 
     if (!shouldExecuteIfOff(endpoint, commandId, optionsMask, optionsOverride))
     {
-        status = Status::Success;
         goto send_default_response;
     }
 
     // Cancel any currently active command.
     cancelEndpointTimerCallback(endpoint);
+    SetCurrentLevelQuietReport(endpoint, state, state->quietCurrentLevel.value(), true /*isStartOrEndOfTransition*/);
```

Comment:
So this one is a bit complicated: if we are in fact at start of transition, but the the current value of quietCurrentLevel has not been reported, then we should in fact report here (right?), and I don't think we will given the code above.

It looks like we need a MarkAttributeDirty::kYes value, probably?